Dockerfile

```diff
@@ -1,46 +1,47 @@
-FROM alpine:3.13 as builder
-LABEL version="1.0" description="OwnTracks Recorder"
-LABEL authors="Jan-Piet Mens <jpmens@gmail.com>, Giovanni Angoli <juzam76@gmail.com>, Amy Nagle <kabili@zyrenth.com>, Malte Deiseroth <mdeiseroth88@gmail.com>"
-MAINTAINER Malte Deiseroth <mdeiseroth88@gmail.com>
+FROM alpine:3.15 AS builder
 
-# build with `docker build --build-arg recorder_version=x.y.z '
-ARG recorder_version=0.8.8
+ARG RECORDER_VERSION=0.8.8
+# ARG RECORDER_VERSION=master
 
-COPY entrypoint.sh /entrypoint.sh
-COPY config.mk /config.mk
-COPY recorder.conf /etc/default/recorder.conf
-COPY recorder-health.sh /usr/local/sbin/recorder-health.sh
+RUN apk add \
```

Comment:
I note you've removed `--no-cache`. I think somebody explained to me that option reduces the image size. Is that not (no longer) the case/

Reply:
Ah okay, I see. I will add it again. It was causing issues on my side. But I think thats resolved now.